Day-01/typescript/Day1.ts

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+
+const NUMBERS = ["one", "two", "three", "four", "five", "six", "seven", "eight", "nine"];
+
+function main() {
+    let content = fs.readFileSync('../../input.txt', 'utf-8');
```

Comment:
Jetzt gibt es hier eine Inkonsistenz mit der anderen Datei

```suggestion
    let content = fs.readFileSync('input.txt', 'utf-8');
```

Reply:
Kann ich mache, dann müsste die Datei halt im typescript Ordner liegen